playwright-e2e/tests/dsm/kitUploadFlow/blood-and-rna-kit-upload-flow.spec.ts

```diff
@@ -24,7 +24,7 @@ import { saveParticipantGuid } from 'utils/faker-utils';
 import { ParticipantListTable } from 'dsm/component/tables/participant-list-table';
 
 test.describe('Blood & RNA Kit Upload', () => {
-test.skip('Verify that a blood & rna kit can be uploaded @dsm @rgp @functional @upload', async ({ page, request}, testInfo) => {
+  test('Verify that a blood & rna kit can be uploaded @dsm @rgp @functional @upload', async ({ page, request}, testInfo) => {
```

Comment:
not working.

<img width="1271" alt="Screenshot 2023-09-20 at 7 05 06 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/86ab83bb-0d19-410c-b136-f655c45cfb6f">

Reply:
forgot to put an await - should be fixed now